src/Aspire.Hosting/ApplicationModel/Docker/ContainerfileStatements.cs

```diff
@@ -0,0 +1,181 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using System.Text.Json;
+
+namespace Aspire.Hosting.ApplicationModel.Docker;
```

Comment:
Instead of have `IContainerFilestatement/IDockerfileStatement` just have an abstract Dockerfilestatement and have DockerfileStage derive from it, and the other statements.